I always get following error, where the execution then stucks:

```
WARN [plugin]: Error during loading "/drone/src/build/node_modules/karma-phantomjs-launcher" plugin:
  Cannot find module 'phantomjs-prebuilt'
INFO [karma]: Karma v0.12.37 server started at http://localhost:9876/
WARN [launcher]: Can not load "PhantomJS", it is not registered!
  Perhaps you are missing some plugin?
```

--- a/.drone.yml
+++ b/.drone.yml
@@ -1,5 +1,9 @@
 build:
-  backend:
+  jsunit:
+    image: morrisjobke/nextcloud-ci-jsunit:1.0.2
+    commands:
+      - ./autotest-js.sh
+  sqlite:
     image: morrisjobke/nextcloud-ci-php7:1.0
     commands:
       - git submodule update --init

--- a/build/package.json
+++ b/build/package.json
@@ -16,7 +16,7 @@
     "karma-junit-reporter": "*",
     "karma-coverage": "*",
     "karma-phantomjs-launcher": "*",
-    "phantomjs": "*",
+    "phantomjs-prebuilt": "*",
     "jasmine-core": "~2.3.4"
   },
   "engine": "node >= 0.8"

